### PR TITLE
[LLVM][SYCL][E2E] Disable xfail for passing tests

### PR DESCRIPTION
**Pull request by ggojska**
Disabled xfail in following tests:

SYCL :: Matrix/SG32/get_coord_int8_matB.cpp
SYCL :: Matrix/get_coord_int8_matB.cpp
SYCL :: Matrix/joint_matrix_annotated_ptr.cpp

### CODE REVIEW

File: sycl/test-e2e/Matrix/SG32/get_coord_int8_matB.cpp [hunk 1]
```diff
@@ -11,8 +11,6 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
-// XFAIL: *
```
Comment (YuriPlyakhin) [inline sycl/test-e2e/Matrix/SG32/get_coord_int8_matB.cpp:14]: I see xfail was removed for both gpu and cpu. Does the test pass on cpu as well?

File: sycl/test-e2e/Matrix/get_coord_int8_matB.cpp [hunk 2]
```diff
@@ -9,7 +9,6 @@
 
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
-// XFAIL: *
```
Comment (YuriPlyakhin) [inline sycl/test-e2e/Matrix/get_coord_int8_matB.cpp:12]: Same here, does the test pass on cpu?

File: sycl/test-e2e/Matrix/SG32/get_coord_int8_matB.cpp (hunk 1 above)
Comment (dkhaldi) [reply sycl/test-e2e/Matrix/SG32/get_coord_int8_matB.cpp:14]: I don't think the pulldown of OpenCL backend that contains the fix on the CPU happened yet. So we should keep XFAIL:cpu for now.

**Review by YuriPlyakhin — APPROVED** [review]
LGTM

**Comment by YuriPlyakhin** [comment]
@intel/llvm-gatekeepers , all green, please, merge

**Pull request merged**